addon/components/x-range-input.js

```diff
@@ -61,14 +59,13 @@ export default Ember.Component.extend(PropertyBindings, {
   value: 0,
 
   /**
-   * On any `input` event, copy the numeric value of the DOM element
-   * onto the `value` property of the component so that it can be
-   * bound to and from.
+   * On any `input` event, take the component and the element `value` and send
+   * it in an action.
    *
    * @private
    */
-  input: function() {
-    this.set('value', Number(this.get('element.value')).valueOf());
+  input() {
+    this.sendAction('action', this, this.get('element.value'));
```

Comment:
It looks like we're not doing a number conversion anymore, why not?

Also, we probably want to still set the `value` property in addition to sending the action. It should still work with 2-way binding, even though it is discouraged. In other words, like a normal Ember input:

```hbs
<x-range-input min=1 max=11 value=(mut volume)/>
```

should still change the `volume` property